### ISSUE TITLE
Gist link that immediately runs the code

### ISSUE THREAD

**Issue by dtolnay** [integer32llc/rust-playground#134]
I would like to put "Run" links next to all of my code samples. The user shouldn't have to click my "Run" link followed by the playground's "Run" button.

Would it be possible to have a query parameter that means immediately run the gist when you land on the page?

**Comment by shepmaster** [comment]
Yes, it should certainly be possible. It's "just" a matter of chaining together some futures. I should be able to take a poke at this on Wednesday evening.

Any bikeshedding on a potential query parameter name?

**Comment by dtolnay** [comment]
`?gist=a58fc361e02c4c0a08fd99cacd9567d1&run`
